lib/anonymousNaming.js

```diff
@@ -45,6 +47,22 @@ function assignUidToComponentSchemas(doc) {
     }
   }
 }
+
+/**
+ * Assign uid to component parameters schemas
+ * 
+ * @private
+ * @param {AsyncAPIDocument} doc 
+ */
+function assignUidToComponentParameterSchemas(doc) {
+  if (doc.hasComponents()) {
+    for (const [parameterKey, parameter] of Object.entries(doc.components().parameters())) {
+      if (parameter.schema()) {
+        parameter.schema().json()[String(xParserSchemaId)] = parameterKey;
```

Comment:
feels like this entire loop is the same as in line 29, just the place where param is taken from is different. Can we do something about it?

Reply:
Added reusability 😄